docs/README

```diff
@@ -147,6 +147,26 @@ meaning that it is possible to unmap any aligned area and turn it into a hole.
 This is implemented using the "FALLOC_FL_PUNCH_HOLE" "mode" of the
 "fallocate()" system call.
 
+Note: If running on the ZFS file system, the default module parameters
```

Comment:
Did you mean "zfs kernel module parameters" ? Would be good to clarify that you do not mean a python module. Also, some people can have zfs compiled it, right, or this is never possible? In any case, I feel like it is safer to say "ZFS kernel driver" instead.

Reply:
yes sorry i did mean kernel module parameters, will fix it